geode-core/src/main/java/org/apache/geode/internal/cache/DiskInitFile.java

```diff
@@ -336,9 +335,7 @@
 
   private final LongOpenHashSet crfIds;
```

Comment:
I realize these variable names are pre-existing... any chance to make them human understandable...?

Reply:
I'm calling that out of scope.

Reply:
Also, i think crf Ids is the readable form in this case :-)